ui/src/shared/components/TimeMachineQueryBuilder.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+
+import {renderWithRedux} from 'src/mockState'
+import {source} from 'mocks/dummyData'
+import {waitForElement, fireEvent} from 'react-testing-library'
+
+import TimeMachineQueryBuilder from 'src/shared/components/TimeMachineQueryBuilder'
+
+jest.mock('src/shared/apis/v2/queryBuilder')
+
+let setInitialState
+describe('TimeMachineQueryBuilder', () => {
+  beforeEach(() => {
+    setInitialState = state => {
```

Comment:
This probably doesn't need to live in the `beforeEach` hook

Reply:
+ 1